File: back-newsletter/src/main/java/com/back/newsletter/pojo/Todo.java
package com.back.newsletter.pojo;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Todo {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    Long id;

    private String title;

    private String description;

    private String adresse;



    private boolean done;

    public Todo() {
    }

    public Todo(Long id, String title, String description, String adresse, boolean done) {
        this.id = id;
        this.title = title;
        this.description = description;
        this.adresse = adresse;
        this.done = done;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getAdresse() {
        return adresse;
    }

    public void setAdresse(String adresse) {
        this.adresse = adresse;
    }

    public boolean isDone() {
        return done;
    }

    public void setDone(boolean done) {
        this.done = done;
    }

    @Override
    public String toString() {
        return "Todo{" + "id=" + id + ", title=" + title + ", description=" + description + ", adresse=" + adresse + ", done=" + done + '}';
    }

}

File: front-newsletter/src/app/model/Todo.ts
export class Todo{
  id: number;
  title: string;
  description: string;
  done: boolean;

  adresse: string;

  constructor(data: any) {
    this.id = data.id;
    this.title = data.title;
    this.description = data.description;
    this.done = data.done;
    this.adresse = data.adresse;
  }


  toString() {
    return "Todo{" +
      "id=" + this.id +
      ", title='" + this.title + '\'' +
      ", description='" + this.description + '\'' +
      ", done=" + this.done +
      ", adresse='" + this.adresse + '\'' +
      '}';
  }

  serialize() {
    return {
      id: this.id,
      title: this.title,
      description: this.description,
      done: this.done,
      adresse: this.adresse
    }
  }

  deserialize(data: any) {
    this.id = data.id;
    this.title = data.title;
    this.description = data.description;
    this.done = data.done;
    this.adresse = data.adresse;
  }
}
